cmdeploy/src/cmdeploy/tests/online/test_1_basic.py

```diff
@@ -182,6 +182,8 @@ def test_expunged(remote, chatmail_config):
         f"find {chatmail_config.mailboxes_dir} -path '*/tmp/*' -mtime +{outdated_days} -type f",
         f"find {chatmail_config.mailboxes_dir} -path '*/.*/tmp/*' -mtime +{outdated_days} -type f",
     ]
+    outdated_days = int(chatmail_config.delete_large_after) + 1
+    find_cmds.append("find {chatmail_config.mailboxes_dir} -path '*/cur/*' -mtime +{outdated_days} -size +200k -type f")
```

Comment:
isn't this missing the actual `-delete`?

Reply:
The test is supposed to check whether files are still there, not delete them itself :)